packages/terra-clinical-result/src/ClinicalResultBloodPressure.jsx

```diff
@@ -103,6 +108,58 @@ const createDatetimeDisplays = (compareDatetimeDisplays) => {
   return null;
 };
 
+const createSystolicDisplay = (systolic, hideUnit, diastolicUnit, id) => {
+  if (!Object.prototype.hasOwnProperty.call(systolic, 'noData')) {
```

Comment:
This is a pretty verbose logic statement. Is this doing more than just checking if the property exists?

```js
if (!systolic.noData)
```

Reply:
Originally this `if` was to check if the component's `systolic` prop was defined and if not return `ResultError`. With the new way we define things the passed `systolic` object is actually our custom object that adds in a few more trimmed values and booleans but is always defined to avoid having to prepend `systolic &&` to a lot of if statements.  However this custom object might be an empty object if the `systolic` prop wasn't defined.  It's `noData` property always will be defined( and might be false) if the prop was though.

Reply:
Welp ignore my paragraph response, turns out we can just do a ! statement
https://github.com/cerner/terra-clinical/pull/648/commits/bcd9623974696cc69c750cf77514f41a424ac509#diff-db5d557fed702f3b0bcac70aa9a018eaR39